db/schemas/employees.js

```diff
@@ -0,0 +1,60 @@
+db.createCollection('employees', {
+  validator: {
+    $jsonSchema: {
+      bsonType: 'object',
+      required: [
+        'firstName',
+        'lastName',
+        'address1',
+        'city',
+        'country',
+        'wage',
+        'position',
+        'isActive'
+      ],
+      properties: {
+        'firstName': {
+          bsonType: 'string',
+        },
+        'lastName': {
+          bsonType: 'string'
+        },
+        'address1': {
+          bsonType: 'string'
+        },
+        'address2': {
+          bsonType: 'string'
+        },
+        'city': {
+          bsonType: 'string'
+        },
+        'state': {
+          bsonType: 'string'
+        },
+        'country': {
+          bsonType: 'string'
+        },
+        'zipcode': {
+          bsonType: 'string'
+        },
+        'wage': {
```

Comment:
missing
  phone: {type: Number, default: ''},
  email: {type: String, required: true, unique: true},

Reply:
I'll add phone and e-mail.  Phone should be a string please see this stack overflow: https://stackoverflow.com/questions/8284647/how-to-store-phone-numbers-on-mysql-databases